### PR TITLE
Adds Jasmine tests to owner.controller.js

### PR DESCRIPTION
**Pull request by William-Liang808**
https://www.hawaii.edu/jira/browse/GROUPINGS-1086

[Groupings-1086](https://www.hawaii.edu/jira/browse/GROUPINGS-1086)

# Test Checklist

- [x ] Unit Tests Passed:
- [x ] Jasmine Tests Passed:
- [x ] General Visual Inspection:

### CODE REVIEW

**Comment by William-Liang808** [comment]
Only codacy test failing is the 'inject' not being defined

**Comment by mhodgesatuh** [comment]
Have you considered this syntax?
/*global inject*/
As mentioned by the codacy examples?

**Comment by William-Liang808** [comment]
> Have you considered this syntax? /_global inject_/ As mentioned by the codacy examples?

Right let me try again, I think may have done this for my previous test pushed.

**Review by devgav — APPROVED** [review]
These changes seem fine. Thank you for helping to increase the code coverage of our javasacript code.

**Comment by devgav** [comment]
Seems like removing the 'a' from the global inject statement still ignores the codacy issue. I can go ahead and merge this changes now.

**Comment by William-Liang808** [comment]
-Rebased and pushed recent UI merges
